plugins/aws/elasticache/elasticacheClusterInVpc.js

```diff
@@ -0,0 +1,55 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'ElastiCache Cluster In VPC',
+    category: 'ElastiCache',
+    domain: 'Databases',
+    description: 'Ensure that your ElastiCache clusters are provisioned within the AWS EC2-VPC platform instead of EC2-Classic platform .',
+    more_info: 'AWS ElastiCache clusters using EC2-VPC platform instead of EC2-Classic can bring multiple advantages such as better networking infrastructure, much more flexible control over access security .',
```

Comment:
```suggestion
    more_info: 'Amazon ElastiCache clusters using EC2-VPC platform instead of EC2-Classic can bring multiple advantages such as better networking infrastructure, much more flexible control over access security .',
```